suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -446,3 +446,22 @@ export const getTestnets = (debug = false) =>
     networksCompatibility.filter(
         n => !n.accountType && n.testnet === true && (n.symbol !== 'regtest' || debug),
     );
+
+export const isNetworkSymbol = (symbol: string): symbol is NetworkSymbol => symbol in networks;
+
+type GetNetwork = {
+    <T extends NetworkSymbol>(
+        symbol: T,
+        accountType?: (keyof Networks[T]['accountTypes'] & string) | 'normal',
+    ): Network;
+    (symbol: string, accountType?: string): Network | null;
+};
+
+export const getNetwork: GetNetwork = (symbol: string, accountType = 'normal') => {
```

Comment:
Played little bit with types and it seems to work https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAYwgOwM72cGB3aBrVOAXjgG8AoOKuAehrgCEBLGJJ5S6gI1YC4zO1asgCGAW2D8A5M1YR2UgDSChcTDnwAVAJ5hJcKVxZtkSlUKNgALAGYACiJgALfgCIxNABxWpNAAy+TFKuyqrUACbACExiIgA2qPyeoWFwoGBx0MBQ-BSpqjAg0k4wMGCJdDwIAIwAdDBQwABe0LVMEDSFNGb5QiIISACuyDDFpeW8lax1Dc2t7TTgg1zdKWEAvmuqAGbAjoONiXAA2lJQXNtKBqhMAObIALQAbtlM29pXUuIQwzAPwywpABdLZCBCDdAQMQMfp4YDIcJHU5cTIIPBcCAQPCfYBxKINQZiYGg6j9IYjHR6I55XpUGAiMBQTGjAS0sKiCTSWQmOAACk0DKZEBgAEoemyLExrPZHC44O4vAA2XwBGhBELmNm7faHfinc6XRQGb6-f7IQEgzUbElhVDAW7YFi5K29Dn6GTGeTIPkAGXt-W0cAAyvbHaLxRKqJZbA5nG4PFYAJwqwLBG2qTYuoR424B52R4Tid3cr2+-0IbRi9P5aMyuPyhM+fypjUSzP5dvUTtUOhwH0sKJelRxPis1Ju6T9mCDhTp9S4KB4SnuoxyWcu2uxuUK7y+ABMLfTkWisQSSXT6UyjRyY9phTGZQqNBHNXqjRaUDaHS6Ed6ZJ+IwPhMdAvjM77zB0SwrL+QjdjsewwAcwBIlINz3M8rzvMSLrgpC0KwvCiJ6oYqLopi2KWvk-6-Mu1JZtQdoOk6t6RhOBhTjO3q8oxYZVvRYSbrK8Y0Em+6HvxcBwfkOZ5ixEpsVIHE8ryMkVnxBZRlKMZCQ2IlNgeapphJUldlscG9gAos42TAISKhYHKNLskW0hWU4NmEjBwhYAuS66O6DkeUS6YIE4IjsAAkuE-DVOmgn1gqVhNoqqqpWqrapMeMTxEc1TJC6l5ZDeTm9PeBglI+kw0A5YFzJ+Cw-tWpIDABLJSBVwHVc4tUfl+NAiOE4SHKgqwuiZVDaohuonKhdyPC8UBvB8RpSDAWLwqg2HSSIXC4tImgAEoAPoAHIWZoADqADyB0ANJHedAASFkHRZACqACyR0+gAggwFk+l5dKYiOUr7cdZ2XTd91PS971fZoV1XT6mgRXYQOIBCa34WihEoSiEBohiWJbak1EUv51Jwes5DrHAIiEEgaAwOQ5BM+gcC3FgJ0+fgxBwAAPJot3AIGoDTgihBwtoEDbHAMD+bLai84uqAAHzcdoYgYnE-DC6LRrkzAy78NLSsK3oSvzvgqDHPr2hAqcRu0cCcAAD4GMg0CnlI-NSF7UA+yKxBq3J7PwKQ9MtTRlNGrU8fW4ukn84nBDHKgWs60CADc5jh3AEALYtkT87yzuU-ThAiMglbHOXeg5+Ybx8oX2TF8AweNFN3qRxn2sQHEhvRxTehxwnKt4GPtSt1A7eSbnqjN2Xw-G-5xBECQ-ve-EUid1gBw93Afc61Pqfz+YXcH2ogxxHEuc0+QvZXbd5BczAPMaIuvKuFUITyvSjJmSuBFKzXs2xwpxFftzCe38Xx-1cAAoUMBgGgPoOApgkC34f18t-By8CeIsBQY-egz8oHvxga4PBIogA